library/std/src/sync/lazy_lock.rs

```diff
@@ -103,23 +157,23 @@ impl<T: Default> Default for LazyLock<T> {
 #[unstable(feature = "once_cell", issue = "74465")]
 impl<T: fmt::Debug, F> fmt::Debug for LazyLock<T, F> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("Lazy").field("cell", &self.cell).finish_non_exhaustive()
+        match self.get() {
+            Some(v) => f.debug_tuple("LazyLock").field(v).finish(),
+            None => f.write_str("LazyLock(Uninit)"),
+        }
```

Comment:
Nit: With a `#[derive(Debug)] struct Uninit;`, a fully initialized `LazyLock<Uninit>` would also produce `"LazyLock(Uninit)"`. Maybe it's better to show a nested `Option` (`LazyLock(Some(..))`), or to use `LazyLock(<uninitialized>)` for the uninitialized case.

Reply:
Oh I see we also already format an empty `OnceCell` as `OnceCell(Uninit)`. So no need to change that in this PR.